src/views/CampusSafety/index.tsx

```diff
@@ -26,7 +26,15 @@ const CampusSafety = () => {
               titleTypographyProps={{ variant: 'h4' }}
             />
             <CardContent>
-              <Typography variant="body1">Gordon Police Resources</Typography>
+              <Link
+                href="https://www.gordon.edu/police"
```

Comment:
Before we commit this, do you also want to add the police phone numbers to this card, maybe add some simple styling to it, but don't worry about getting it to look 100% yet.